### ISSUE TITLE
Per company lists

### ISSUE THREAD

**Issue by ai** [davydovanton/ossboard#31]
@gaserd find a nice idea for his OS board — allow companies to have own lists of task. So it will be like filter. We have all tasks on main page. But have a “editor choice” by some company.

It could be a good promotion for other companies to create a tasks (even to side projects). Also users like “editor choice” like we see in Cult of Martians.

**Comment by davydovanton** [comment]
@ai now we have repository filter. 'Company' list is a good idea, but I think we need to use 'organisation' name here.

**Comment by ai** [comment]
Sure. Organization is better.

**Comment by ai** [comment]
But it is about repository. Organization could promote different repositories (even not own). Like we do in the Cult. Also having maybe some branding will be nice for this organizations pages.

When you have big lost (like iTunes or Google Play). Common filter UI is bad. But some small groups like "Scary movies", "This year Oscar" are much better. 

So organizition tasks groups should not be just a select on top of list. It should be separated page and maybe some block in main list (check iTunes or Google Play).

**Comment by davydovanton** [comment]
Yep, agree with you. I think it will be helpful for organizations like rom, hanami, dry, grape, Babel, etc. Because these organizations have 7+ different repositories and it will strange if we do not group the repositories in one organization.

Also, I believe we can use a unique color block or something like this for each organization. 

What do you think @DaZzz?

**Comment by DaZzz** [comment]
Sounds good to me. Still need to figure out how to do _branding_ properly. I kinda like how organisations can tune their medium blog appearance to align it with brand guidelines. Although I don't know how it works honestly. 

I will investigate more on this issue. Still want to do redesign of existing pages first though. 😅
